packages/vscode-ui5-language-assistant/.vscodeignore

```diff
@@ -12,11 +12,19 @@ node_modules
 !**/@ui5-language-assistant/language-server/src/**
 
 # The VSCode Extension's dev artifacts
+src/**
+lib/**/*.js.map
+dist/**/*.js.map
+
 coverage/**
 .nyc_output/**
 lib/test/**
 test/**
 scripts
+nyc.config.js
```

Comment:
if you are going to the level of very specific excluding of all files.

It may be easier to maintain long-term if you:
- exclude everything.
- include only what is needed (e.g package.json `lib` folder, other?)